mohawk/util.py

```diff
@@ -268,3 +267,28 @@ def normalize_header_attr(val):
     # Normalize like the hawk reference code.
     val = escape_header_attr(val)
     return val
+
+
+def get_bewit(resource):
+    """Returns a bewit identifier for the resource as a string"""
+    assert resource.method == 'GET'
+    mac = calculate_mac(
+        'bewit',
+        resource,
+        None,
+    )
+
+    if isinstance(mac, six.binary_type):
+        mac = mac.decode('ascii')
+
+    if resource.ext is None:
+        ext = ''
+    else:
+        ext = resource.ext
+
+    inner_bewit = u"{id}\\{exp}\\{mac}\\{ext}".format(
+        id=resource.credentials['id'],
+        exp=resource.timestamp,
+        ext=ext,
+        mac=mac).encode('ascii')
+    return urlsafe_b64encode(inner_bewit).decode("ascii")
```

Comment:
ok, we caught up on irc. I think it would be nicer to make this a bit more explit (perhaps with some comments about Python 2 vs 3 compat). Something like:
````
inner_bewit = u"{id}\\{exp}\\{mac}\\{ext}".format(...)
inner_bewit_bytes = inner_bewit.encode('ascii')
bewit = urlsafe_b64encode(inner_bewit_bytes)
return bewit.decode('ascii')
````